packages/cloud_firestore/example/lib/main.dart

```diff
@@ -73,7 +73,7 @@ class MyHomePage extends StatelessWidget {
       floatingActionButton: new FloatingActionButton(
         onPressed: _addMessage,
         tooltip: 'Increment',
-        child: new Icon(Icons.add),
+        child: const Icon(Icons.add),
```

Comment:
In the new dart 2 optional keyword paradigm if the instantiation keyword was not used, would the analyzer infer it was a constant?

Reply:
The analyzer might be able to warn that the expression could be const, but const would need to be declared for the VM to consider it a const expression.